assets/js/payment-method-extensions/payment-methods/stripe/credit-card/index.js

```diff
@@ -3,6 +3,7 @@
  */
 import { __ } from '@wordpress/i18n';
 import { useEffect, useState } from '@wordpress/element';
+import { Icon, card } from '@woocommerce/icons';
```

Comment:
The problem with this is `@woocommerce/icons` is just an alias to internal (non published) components. So we'll have to publish the package and/or export it as an external for when stripe is eventually moved to the payment method extension.

Also wcPay won't be able to use this for it's integration.

I wonder if instead, we should expose a `<PaymentMethodLabel/>` component on the `components` prop provided to the label config and that way Payment methods can use that along with a blessed set of icons to select from?

So something _roughly_ like the following could be passed in via the `components` property:

```jsx
import { card, Icon } from '@woocommerce/icons';

const PaymentMethodLabel = ( { iconType = 'card', text } ) => {
  iconWhitelist = { card };
  if ( iconWhitelist[ iconType ] ) {
    return <span className="payment-method-label-with-icon">
			<Icon srcElement={ iconWhitelist[ iconType ] } />
			{ __( 'Credit / Debit Card', 'woo-gutenberg-products-block' ) }
		</span>
  }
  return text;
}
```

Reply:
Hmm @nerrad this feels like over engineering... Maybe we just use an SVG directly for this until we have suitable public components? Or just expose a common cc icon component only. It's not really clear if other icons would even be useful in this context.

Reply:
Well we _could_ just expose a `CreditCardLabel` component that is very specific to this design and just receives the text content. Then at least any payment methods offering credit cards could just implement this component (with optionally their custom text)?

Reply:
The main thing I'm concerned about here is removing the dependency on the aliased package so we don't have to deal with this when the stripe integration is moved into it's extension.